tests/extensions/extra/tables.txt

```diff
@@ -80,3 +80,13 @@ Lists are not tables
 
  - this | should | not
  - be | a | table
+
+Add tests for issue #449
+
+Odd backtics | Even backtics
```

Comment:
Done.  Commit update worked great, thanks for letting me know.  I've was under the incorrect assumption it GitHub didn't support it. Wish I had know about this sooner :).